onnxruntime/core/providers/nuphar/compiler/x86/scheduler/tensorize/intrin_gemv_8bit.cc

```diff
@@ -59,11 +59,11 @@ tvm::TensorIntrin TensorizeIntGemv8bit::CreateTensorIntrin() {
       0,
       /*offset_factor*/ 1);
 
-  int h_unroll = shape_[1] / 32;
+  int row_unroll = shape_[1] / 32;
   auto sum_int32x8 = tvm::make_const(HalideIR::Int(32, 8), 0);
```

Comment:
>32 [](start = 51, length = 2)

lots of magic numbers in this code. some comments would be helpful, and better to make them into statics with meaningful names.